src/neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -145,7 +153,8 @@ public ContractManifest Clone()
                 Abi = Abi.Clone(),
                 Permissions = Permissions.Select(p => p.Clone()).ToArray(),
                 Trusts = Trusts,
-                SafeMethods = SafeMethods
+                SafeMethods = SafeMethods,
+                Extra = Extra
```

Comment:
`JObject` is mutable. So you need to clone it.

Reply:
@erikzhang  I've noticed we don't have clone on JObject. Do you prefer to add clone on JObject or add the logic only inside this method?

Reply:
Add it in `JObject`.

Reply:
@erikzhang  done